test/testutil/primitives/ticket_utils.hpp

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_TEST_TESTUTIL_PRIMITIVES_TICKET_UTILS_HPP
+#define CPP_FILECOIN_TEST_TESTUTIL_PRIMITIVES_TICKET_UTILS_HPP
+
+#include "primitives/ticket/epost_ticket.hpp"
+#include "primitives/ticket/ticket.hpp"
+
+#include <libp2p/crypto/random_generator/boost_generator.hpp>
+
+/**
+ * @class TicketGenerator provides methods for creating random tickets and
+ * related objects
+ */
+class TicketGenerator {
```

Comment:
See [comment for #42](https://github.com/filecoin-project/cpp-filecoin/pull/42#discussion_r360801059) for possible reasons to remove random generation from tests.

Reply:
Agree with @turuslan.

Reply:
I agree that we shouldn't rely only on random generated data, but such data is useful to cover cases not affordable otherwise. The argument about debug difficulties should be taken into account, I provide test with more debug information.